packages/node-cli/src/services/SuiteCloudAuthProxyService.js

```diff
@@ -158,6 +184,16 @@ class SuiteCloudAuthProxyService extends EventEmitter {
 		}
 	}
 
+	/**
+	 * Matches /api/internal/devassist/ followed by one or more valid URL path characters
+	 * @param path
+	 * @returns {boolean}
+	 * @private
+	 */
+	_requestPathPatternValid(path) {
+		return path.startsWith(this._allowedPathPrefix);
+	}
+
```

Comment:
I don't think we should mention devassist in the comments. 
The parameter could be named `requestPath` to make it clear we are talking about request paths and not fileSystem paths.

Reply:
Another think about this validation method, let's write in a way it will work if `_allowedPathPrefix` is undefined.